source/frontend/src/utils/mapPropertyUtils.ts

```diff
@@ -144,3 +147,59 @@ function toMapProperty(
     name: feature?.properties?.NAME,
   };
 }
+
+export function featuresetToMapProperty(
+  featureSet: LocationFeatureDataset,
+  address: string = 'unknown',
+): IMapProperty {
+  const pimsFeature = featureSet.pimsFeature;
+  const parcelFeature = featureSet.parcelFeature;
+  const regionFeature = featureSet.regionFeature;
+  const districtFeature = featureSet.districtFeature;
+
+  const propertyId = pimsFeature?.properties.PROPERTY_ID;
```

Comment:
i don't know enough to know if properties can be null/undefined.

Reply:
I dont think it can